sdk/tables/azure-data-tables/azure/data/tables/_constants.py

```diff
@@ -5,7 +5,7 @@
 # --------------------------------------------------------------------------
 import platform
 import sys
-from ._generated.version import VERSION
+from ._generated._version import VERSION
 
 # UserAgent string sample: 'Azure-Storage/0.37.0-0.38.0 (Python CPython 3.4.2; Windows 8)'
 # First version(0.37.0) is the common package, and the second version(0.38.0) is the service package
```

Comment:
It doesn't look like that user agent code is being used, instead we're setting it in base client. @seankane-msft, can you remove this constant?